src/core/internal/parseoptions.d

```diff
@@ -168,6 +168,7 @@ inout(char)[] find(alias pred)(inout(char)[] str)
 }
 
 bool parse(T : size_t)(const(char)[] optname, ref inout(char)[] str, ref T res, const(char)[] errName, bool mayHaveSuffix = false)
+if (is(T == size_t))
```

Comment:
Why make it a template at all if there can be only one instantiation of it?

Reply:
I'd just replace size_t with ulong to make it portable.

Reply:
Because this PR only reduces code size, it doesn't also refactor or change signatures.

Reply:
> I'd just replace size_t with ulong to make it portable.

That won't work either, as the T parameter is `ref T res`.